<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Bansal Lab - Publications</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/heroic-features.css" rel="stylesheet">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation -->
    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                 <a class="navbar-brand" href="index.html">Bansal Lab</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="research.html">Research</a>
                    </li>
                    <li>
                        <a href="publications.html">Publications</a>
                    </li>
                    <li>
                        <a href="people.html">People</a>
                    </li>
                    <li>
                        <a href="teaching.html">Teaching</a>
                    </li>
                    <li>
                        <a href="software.html">Software</a>
                    </li>
                    <li>
                        <a href="data.html">Data</a>
                    </li>
                    <li>
                        <a href="join.html">Join</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Page Content -->
    <div class="container">

        <!-- Jumbotron Header -->
        <header class=jumbotron>
            <h4><a href="https://scholar.google.com/citations?hl=en&user=B4b2F7cAAAAJ&view_op=list_works&sortby=pubdate">Google Scholar</a> <br><br></h4>
                
	    <div><p style="font-size:14px;font-weight:350;"><b><font face="arial, sans-serif">2021-2022<br><br></b>
					    Prodanov T, Bansal V. Robust and accurate estimation of paralog-specic copy number for
					    duplicated genes using whole-genome sequencing. to appear in Nature Communications. <a href="https://github.com/tprodanov/parascopy"> [Parascopy software]</a> <a href="slides/parascopy-ismbslides.pdf">[ISMB slides pdf]</a><br><br> 

					    Bansal V. HapCUT2: a method for phasing genomes using experimental sequence data. Book chapter in Methods in Molecular Biology (to appear), editors Brock Peters and Radoje Drmanac. <a href="https://github.com/vibansal/hapcut2"> [HapCUT2 software]</a> <br><br> 
					    
					    Wagner J, Olson N, Harris L et al. <a href="https://www.sciencedirect.com/science/article/pii/S2666979X2200057X">Benchmarking challenging small variants with linked and long reads.</a>Cell Genomics, Vol 5(2), 2022 <br><br>


                <!--<a href="publications-all.html">Papers by chronological order</a><br><br> -->
			<div><p style="font-size:14px;font-weight:350;"><b><font face="arial, sans-serif">2020<br><br></b>

			Prodanov T, Bansal V. Sensitive alignment using paralogous sequence variants improves long read mapping and variant calling in segmental duplications. <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa829/5920427">Nucleic Acids Research, 2020</a> <a href="https://gitlab.com/tprodanov/duplomap"> [Duplomap software]</a> <a href="slides/duplomap-slides.pdf">[slides PDF]</a> <br><br> 

					    Chin C, Wagner J, Zeng Q et al. <a href="https://www.nature.com/articles/s41467-020-18564-9">A Diploid Assembly-based Benchmark for Variants in the Major Histocompatibility Complex</a>. Nature Communications 11, 2020. <br><br>

Chen Z, Pham L, Wu TC et al. <a href="https://genome.cshlp.org/content/30/6/898.long">Ultra-low input single tube linked-read library method enables short-read NGS systems to generate highly accurate and economical long-range sequencing information for de novo genome assembly and haplotype phasing</a> Genome Research 30: 898-909, June 2020. <br><br>
				    </p>
				    <div><p style="font-size:14px;font-weight:350;"><b><font face="arial, sans-serif">2019<br><br></b>
				    
Edge P, Bansal V. <a href="https://www.nature.com/articles/s41467-019-12493-y">Longshot: accurate variant calling in diploid genomes using single-molecule long read sequencing.</a> Nature Communications, Oct 2019. <a href="https://github.com/pjedge/longshot"> [software download]</a><br><br>   

Scott ER, Bansal V, Meacham C, Scott SA. <a href="https://doi.org/10.1016/j.jmoldx.2019.10.005">VarCover: Allele Min-Set Cover Software.</a>The Journal of Molecular Diagnostics, 2019. <a href="https://varcover.org"> [software] </a> <br><br>
				    
Bansal V and Boucher C.  <a href="https://www.sciencedirect.com/science/article/pii/S2589004219302184">Sequencing Technologies and Analyses: Where Have We Been and Where Are We Going?</a> iScience, Aug 2019.  <br><br>   
				    
Bansal V. <a href="https://academic.oup.com/bioinformatics/article/35/14/i242/5529122?guestAccessKey=cfe9a78f-f234-4099-bdd8-12cab01556cf">Integrating read-based and population-based phasing for dense and accurate haplotyping of individual genomes.</a> In proceedings of <a href="https://www.iscb.org/ismbeccb2019">ISMB/ECCB 2019</a> (published in Bioinformatics, July 2019). <a href="slides/Bansal-HitSeq2019.pdf">[ISMB slides pdf]</a> <br><br>
				    </p>
		
				    <div><p style="font-size:14px;font-weight:350;"><b><font face="arial, sans-serif">2018<br><br></b>
                
<b>Bansal V</b>, Boehm BO, Darvasi A. <a href="https://link.springer.com/article/10.1007/s00125-018-4690-3">Identification of a rare missense mutation in the WFS1 gene that causes a mild form of Wolfram syndrome and is associated with risk of type 2 diabetes in Ashkenazi Jewish</a>.  Diabetologica, July 2018. <a href="https://rdcu.be/25Tc">[view PDF]</a> <br><br>
                
Bakhtiari M, Shleizer-Burko S, Gymrek M, <b>Bansal V</b>, Bafna V. <a href="https://genome.cshlp.org/content/28/11/1709">Targeted Genotyping of Variable Number Tandem Repeats with
				    adVNTR.</a> Genome Research, October 2018. <br><br>
	</p>
                
	<div><p style="font-size:14px;font-weight:350;"><b><font face="arial, sans-serif">2017<br><br></b> 
                
<b>Bansal V</b>, Gassenhuber J, Phillips T, et al. <a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-017-0977-3">Spectrum of mutations in monogenic diabetes genes identified from high-throughput DNA sequencing of 6,888 individuals.</a> <b>BMC Medicine</b>, 2017 <a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-018-1012-z">Commentary in BMC Medicine</a> <br><br>
                        
Chu WK, Edge P, Lee HS, <b>Bansal V</b>, Bafna V, Huang X, Zhang K. <a href="http://www.pnas.org/content/early/2017/10/23/1707609114.abstract">Ultra Accurate Genome Sequencing And Haplotyping Of Single Human Cells</a>. PNAS, October 2017.<a href="http://jacobsschool.ucsd.edu/news/news_releases/release.sfe?id=2355">UCSD News article</a><br><br>
                        
<b>Bansal V</b>. <a href="https://academic.oup.com/bioinformatics/article/doi/10.1093/bioinformatics/btx436/3952670/An-accurate-algorithm-for-the-detection-of-DNA?guestAccessKey=31be2712-1ab0-4639-a204-c0a209701d34">An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments</a>. Bioinformatics 2017. Also presented at the <a href="http://cb.csail.mit.edu/cb/recomb2017/recomb-seq.html">RECOMB-Seq 2017 conference</a>. <br>
    
                <div>Tiwari S, Dang J, Qin Y, Lichinchi G, <b>Bansal V</b>, Rana TM. <a href="http://www.nature.com/emi/journal/v6/n4/full/emi20179a.html">Zika virus infection reprograms global transcription of host cells to allow sustained infection</a><span style="font-family:arial,sans-serif;line-height:20px;background-color:transparent">. Emerging Microbes and Infection, April 2017.&nbsp;</span></div><br>
    
<b>Bansal V</b>. <a href="http://rdcu.be/p6Iq">A computational method for estimating the PCR duplication rate in high-throughput DNA sequencing experiments</a>. BMC Bioinformatics, March 2017.  <a href="https://github.com/vibansal/PCRduplicates">[code]</a> <br>
<br>
		</p>
<div><b style="line-height:20px"><font face="arial, sans-serif" size="2">2015-2016<br></p></b></div><br>

<div><p style="font-size:14px;font-weight:350;">Edge P, Bafna V, <b>Bansal V</b>. <a href="http://genome.cshlp.org/content/early/2016/12/09/gr.213462.116.abstract">HapCUT2: robust and accurate haplotype assembly for diverse sequencing technologies</a>, Genome Research, Dec 2016. <a href="https://github.com/pjedge/hapcut2">[software link]</a>&nbsp;<br><br>
        

Patel A, Edge P, Selvaraj S, <b>Bansal V </b>and Bafna V. <a href="http://nar.oxfordjournals.org/content/early/2016/04/21/nar.gkw281.abstract"> InPhaDel: Integrative shotgun and proximity-ligation sequencing to phase deletions with single nucleotide polymorphisms</a>.  Nucleic Acids Research, 2016. <a href="https://github.com/l337x911/inphadel">[software link]</a><br>
<br>
Lichinchi G, Gao S, Saletore Y, Gonzalez GM, <b>Bansal V</b>, Wang Y, Mason CE, Rana TM. <a href="http://www.nature.com/articles/nmicrobiol201611">Dynamics of the human and viral m6A RNA methylomes during HIV-1 infection of T cells</a>. Nature Microbiology, February 2016<br>
<br>
<b>Bansal V,</b> Libiger O. <a href="http://www.biomedcentral.com/1471-2105/16/4">Fast individual ancestry inference from DNA sequence data leveraging allele frequencies for multiple populations</a>. BMC Bioinformatics, Jan 2015  <a href="software/iadmix.html">[software]</a> <a href="research/iadmix-slides-github.pdf">[SLIDES]</a> (Note: In Table 1, the column headings for European and African are reversed)</p><br>
</p></div>
            <h4><a href="olderpubs.html">publications from 2014 and before</a> <br><br></h4>
            <!-- <p>Publications goes here .. </p> -->
        </header>
    </div>
    <!-- /.container -->

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

</body>

</html>
