<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Bansal Lab - Publications</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/heroic-features.css" rel="stylesheet">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation -->
    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                 <a class="navbar-brand" href="index.html">Bansal Lab</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="research.html">Research</a>
                    </li>
                    <li>
                        <a href="publications.html">Publications</a>
                    </li>
                    <li>
                        <a href="people.html">People</a>
                    </li>
                    <li>
                        <a href="teaching.html">Teaching</a>
                    </li>
                    <li>
                        <a href="software.html">Software</a>
                    </li>
                    <li>
                        <a href="data.html">Data</a>
                    </li>
                    <li>
                        <a href="join.html">Join</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Page Content -->
    <div class="container">

        <!-- Jumbotron Header -->
        <header class=jumbotron>
            <h4><a href="https://scholar.google.com/citations?hl=en&user=B4b2F7cAAAAJ&view_op=list_works&sortby=pubdate">Google Scholar</a> <br><br></h4>
                
                <!--<a href="publications-all.html">Papers by chronological order</a><br><br> -->

		            <div><font size="2"><b><font face="arial, sans-serif">2019-<br><br></font></b>
				    
Prodanov T, Bansal V. Accurate mapping of long SMS reads to segmental duplications in the human genome. In preparation <br><br> 

Chen Z, Pham L,..., Bansal V, ...,Lei M. <a href="https://www.biorxiv.org/content/10.1101/852947v1">Ultra-low input single tube linked-read library method enables short-read NGS systems to generate highly accurate and economical long-range sequencing information for de novo genome assembly and haplotype phasing</a> bioRxiv preprint, November 2019. <br><br>
				    
Edge P, Bansal V. <a href="https://www.nature.com/articles/s41467-019-12493-y">Longshot: accurate variant calling in diploid genomes using single-molecule long read sequencing.</a> Nature Communications, Oct 2019. <a href="https://github.com/pjedge/longshot"> [software download]</a><br><br>   

Scott ER, Bansal V, Meacham C, Scott SA. <a href="https://doi.org/10.1016/j.jmoldx.2019.10.005">VarCover: Allele Min-Set Cover Software.</a>The Journal of Molecular Diagnostics, 2019. <a href="https://varcover.org"> [software] </a> <br><br>
				    
Bansal V and Boucher C.  <a href="https://www.sciencedirect.com/science/article/pii/S2589004219302184">Sequencing Technologies and Analyses: Where Have We Been and Where Are We Going?</a> iScience, Aug 2019.  <br><br>   
				    
Bansal V. <a href="https://academic.oup.com/bioinformatics/article/35/14/i242/5529122?guestAccessKey=cfe9a78f-f234-4099-bdd8-12cab01556cf">Integrating read-based and population-based phasing for dense and accurate haplotyping of individual genomes.</a> In proceedings of <a href="https://www.iscb.org/ismbeccb2019">ISMB/ECCB 2019</a> (published in Bioinformatics, July 2019). <br><br>
		
<div><font size="2"><b><font face="arial, sans-serif">2018<br><br></font></b>
                
<b>Bansal V</b>, Boehm BO, Darvasi A. <a href="https://link.springer.com/article/10.1007/s00125-018-4690-3">Identification of a rare missense mutation in the WFS1 gene that causes a mild form of Wolfram syndrome and is associated with risk of type 2 diabetes in Ashkenazi Jewish</a>.  Diabetologica, July 2018. <a href="https://rdcu.be/25Tc">[view PDF]</a> <br><br>
                
Bakhtiari M, Shleizer-Burko S, Gymrek M, <b>Bansal V</b>, Bafna V. <a href="https://genome.cshlp.org/content/28/11/1709">Targeted Genotyping of Variable Number Tandem Repeats with
				    adVNTR.</a> Genome Research, October 2018. <br><br>
                
 <div><font size="2"><b><font face="arial, sans-serif">2017<br><br></font></b> 
                
<b>Bansal V</b>, Gassenhuber J, Phillips T, et al. <a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-017-0977-3">Spectrum of mutations in monogenic diabetes genes identified from high-throughput DNA sequencing of 6,888 individuals.</a> <b>BMC Medicine</b>, 2017 <a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-018-1012-z">Commentary in BMC Medicine</a> <br><br>
                        
<div><font face="arial, sans-serif" size="2">Chu WK, Edge P, Lee HS, <b>Bansal V</b>, Bafna V, Huang X, Zhang K. <a href="http://www.pnas.org/content/early/2017/10/23/1707609114.abstract">Ultra Accurate Genome Sequencing And Haplotyping Of Single Human Cells</a>. PNAS, October 2017.</font> <a href="http://jacobsschool.ucsd.edu/news/news_releases/release.sfe?id=2355">UCSD News article</a> </div><br>
                        
<div><font face="arial, sans-serif" size="2"><b>Bansal V</b>. <a href="https://academic.oup.com/bioinformatics/article/doi/10.1093/bioinformatics/btx436/3952670/An-accurate-algorithm-for-the-detection-of-DNA?guestAccessKey=31be2712-1ab0-4639-a204-c0a209701d34">An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments</a>. Bioinformatics 2017. Also presented at the <a href="http://cb.csail.mit.edu/cb/recomb2017/recomb-seq.html">RECOMB-Seq 2017 conference</a>. </font></div><br>
    

                <div>Tiwari S, Dang J, Qin Y, Lichinchi G, <b>Bansal V</b>, Rana TM. <a href="http://www.nature.com/emi/journal/v6/n4/full/emi20179a.html">Zika virus infection reprograms global transcription of host cells to allow sustained infection</a><span style="font-family:arial,sans-serif;line-height:20px;background-color:transparent">. Emerging Microbes and Infection, April 2017.&nbsp;</span></div><br>
    
<div> <b>Bansal V</b>. <a href="http://rdcu.be/p6Iq">A computational method for estimating the PCR duplication rate in high-throughput DNA sequencing experiments</a>. BMC Bioinformatics, March 2017.  <a href="https://github.com/vibansal/PCRduplicates">[code]</a> </div><br>
<br>
    
<div><b style="line-height:20px"><font face="arial, sans-serif" size="2">2015-2016<br></font></b></div><br>

<div><font size="2"><font face="arial, sans-serif">Edge P, Bafna V, <b>Bansal V</b>. <a href="http://genome.cshlp.org/content/early/2016/12/09/gr.213462.116.abstract">HapCUT2: robust and accurate haplotype assembly for diverse sequencing technologies</a>, Genome Research, Dec 2016. <a href="https://github.com/pjedge/hapcut2">[software link]</a>&nbsp;<br><br>
        

Patel A, Edge P, Selvaraj S, <b>Bansal V </b>and Bafna V. <a href="http://nar.oxfordjournals.org/content/early/2016/04/21/nar.gkw281.abstract"> InPhaDel: Integrative shotgun and proximity-ligation sequencing to phase deletions with single nucleotide polymorphisms</a>.  Nucleic Acids Research, 2016. <a href="https://github.com/l337x911/inphadel">[software link]</a><br>
<br>
Lichinchi G, Gao S, Saletore Y, Gonzalez GM, <b>Bansal V</b>, Wang Y, Mason CE, Rana TM. <a href="http://www.nature.com/articles/nmicrobiol201611">Dynamics of the human and viral m6A RNA methylomes during HIV-1 infection of T cells</a>. Nature Microbiology, February 2016<br>
<br>
<b>Bansal V,</b> Libiger O. <a href="http://www.biomedcentral.com/1471-2105/16/4">Fast individual ancestry inference from DNA sequence data leveraging allele frequencies for multiple populations</a>. BMC Bioinformatics, Jan 2015  <a href="software/iadmix.html">[software]</a> <a href="research/iadmix-slides-github.pdf">[SLIDES]</a> (Note: In Table 1, the column headings for European and African are reversed)</font><br>
<br>
<br>
</font></div>
<div><b><font face="arial, sans-serif" size="2">2013-2014</font></b></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div></div>
<div><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px">Scott-Van Zeeland AA, Bloss C, Tewhey C, <b>Bansal V</b>, et al “</span><a href="http://www.nature.com/mp/journal/vaop/ncurrent/full/mp201391a.html" style="color:rgb(51,51,51);line-height:22.2222px;background-image:initial;background-color:rgb(231,231,231);background-repeat:initial">Evidence for the Role of EPHX2 gene variants in Anorexia Nervosa</a><span style="line-height:22.2222px">”.&nbsp;&nbsp; Molecular&nbsp; Psychiatry, 2014</span><br style="line-height:22.2222px">
</font></div>
<div><font face="arial, sans-serif" size="2"><br>
</font>
<div>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px;background-color:transparent">Selvaraj S, Dixon JR,<b> Bansal V</b>, Ren B.</span><span style="line-height:22.2222px;background-color:transparent">&nbsp;</span><a href="http://www.nature.com/nbt/journal/vaop/ncurrent/abs/nbt.2728.html" style="line-height:22.2222px">Whole genome haplotype reconstruction using proximity-ligation and shotgun sequencing</a><span style="line-height:22.2222px;background-color:transparent">. Nature Biotechnology 2013</span><span style="line-height:22.2222px;background-color:transparent"> <br>
</span></font></div>
</div>
</div>
<div><span style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
<div><b><font face="arial, sans-serif" size="2">2011-2012</font></b></div>
<div><span style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
<div><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px">Torkamani A, Pham P, Libiger O, <b>Bansal V</b>, Zhang G, Scott-Van Zeeland AA, Tewhey R, Topol EJ, Schork NJ.&nbsp;</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/23125845" style="line-height:22.2222px;background-color:rgb(255,255,255)">Clinical implications of human population differences in genome-wide rates of functional genotypes.</a><span style="line-height:22.2222px">&nbsp;</span><span style="line-height:22.2222px" title="Frontiers in genetics">Front Genet</span><span style="line-height:22.2222px">. 2012;3:211.</span></font>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><br>
</font></div>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2">Tewhey R, Cannavino CR, Leake JA, <b>Bansal V</b>, Topol EJ, Torkamani A, Bradley JS, Schork NJ.&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/23009684">Genetic structure of community acquired methicillin-resistant Staphylococcus aureus USA300.</a>&nbsp;<span title="BMC genomics">BMC Genomics</span>. 2012 Sep 25;13:508.&nbsp;</font></div>
</div>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><br>
</font></div>
<div style="line-height:22.2222px">
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><b>Bansal V</b>, Libiger O.&nbsp;<a href="http://bioinformatics.oxfordjournals.org/content/27/15/2047.abstract" style="color:rgb(51,51,51);background-image:initial;background-color:rgb(255,255,255);background-repeat:initial">A probabilistic method for the detection and genotyping of small indels from population-scale sequence data</a>. Bioinformatics. 2011 Aug 1;27(15):2047-53. Epub 2011 Jun 7.&nbsp;</font></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px"><b>Bansal V</b>, Libiger O, Torkamani A, Schork NJ.&nbsp;</span><a href="http://psb.stanford.edu/psb-online/proceedings/psb11/bansal.pdf" style="line-height:22.2222px;background-color:rgb(255,255,255)">An application and empirical comparison of statistical analysis methods for associating rare variants to a&nbsp;complex phenotype</a><span style="line-height:22.2222px">. Pac Symp Biocomput. 2011:76-87.</span><br style="line-height:22.2222px">
</font></div>
<div><span style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
<div><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px">Lo C, Bashir A, Bansal V, Bafna V.&nbsp;</span><a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S1-S24" style="line-height:22.2222px;background-color:rgb(255,255,255)">Strobe sequence design for haplotype assembly</a><span style="line-height:22.2222px">. BMC Bioinformatics. 2011 Feb 15;12 Suppl 1:S24.</span><br style="line-height:22.2222px">
<br style="line-height:22.2222px">
<span style="line-height:22.2222px">Tewhey R, Bansal V, Torkamani A, Topol EJ, Schork NJ.&nbsp;</span><a href="http://www.nature.com/nrg/journal/v12/n3/abs/nrg2950.html" style="line-height:22.2222px;background-color:rgb(255,255,255)">The importance of phase information for human genomics.</a><span style="line-height:22.2222px">&nbsp;Nature Review Genetics 2011 Mar;12(3):215-23.&nbsp;</span><br style="line-height:22.2222px">
<br style="line-height:22.2222px">
<span style="line-height:22.2222px">Bansal V, Tewhey R, Topol EJ, Schork NJ.&nbsp;</span><a href="http://www.nature.com/nbt/journal/v29/n1/abs/nbt.1757.html" style="line-height:22.2222px;background-color:rgb(255,255,255)">The next phase in human genetics</a><span style="line-height:22.2222px">. Nature Biotechnology 2011 Jan;29(1):38-9.&nbsp;</span></font></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
</div>
<div style="line-height:22.2222px">
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><span style="line-height:1.5">Bansal V, Tewhey R, Leproust EM, Schork NJ.&nbsp;</span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068187/pdf/pone.0018353.pdf" style="line-height:1.5">Efficient and cost effective population resequencing by pooling and in-solution hybridization</a><span style="line-height:1.5">. PLoS One. 2011 Mar 30;6(3):e18353.&nbsp;</span></font></div>
<div><span style="line-height:1.5"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
<div><span style="line-height:1.5"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
</div>
<div style="line-height:22.2222232818604px"><b><font face="arial, sans-serif" size="2">2009-2010</font></b></div>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><br>
</font></div>
<div style="line-height:22.2222px">
<div style="line-height:22.2222px">
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px;background-color:transparent">Bansal V, Libiger O, Torkamani A, Schork NJ.</span><span style="line-height:22.2222px;background-color:transparent">&nbsp;</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/20940738" style="line-height:22.2222px">Statistical analysis strategies for association studies involving rare variants.</a><span style="line-height:22.2222px;background-color:transparent">&nbsp;</span><span style="line-height:22.2222px;background-color:transparent">Nature Reviews Genetics 2010</span></font></div>
<div><span style="line-height:22.2222px;background-color:transparent"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
<font face="arial, sans-serif" size="2">Harismendy O, Bansal V, et al.&nbsp;<a href="https://sites.google.com/">Population sequencing of two endocannabinoid metabolic genes identifies rare and common regulatory variants&nbsp;</a><a href="https://sites.google.com/">associated with extreme obesity and metabolite level</a>. Genome Biology 2010;11(11):R118.&nbsp;<br>
<br>
Bashir A, Bansal V, Bafna V.&nbsp;<a href="http://www.biomedcentral.com/1471-2164/11/385">Designing deep sequencing experiments: detecting structural variation and estimating transcript abundance</a>. BMC Genomics. 2010 June 18;11:385.&nbsp;<a href="http://www.genomeweb.com/sequencing/new-protocols-offer-guidance-optimizing-sequencing-experiments-assess-cnvs-detec">[article on GenomeWeb]</a><br>
<br>
</font></div>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2">Bhatia G, Bansal V, Harismendy O, Schork NJ, Topol EJ, Frazer K, Bafna V.&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2954823/">A covering method for detecting genetic associations between rare variants andcommon phenotypes.</a>&nbsp;PLoS Comput Biol. 2010 Oct 14;6(10):e1000954.&nbsp;</font></div>
</div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div><font face="arial, sans-serif" size="2"><span style="line-height:1.5;background-color:transparent">Bansal V. </span><a href="http://bioinformatics.oxfordjournals.org/content/26/12/i318.long" style="line-height:1.5">A statistical method for the detection of variants from next-generation resequencing of DNA pools</a><span style="line-height:1.5;background-color:transparent">. Bioinformatics. 2010 Jun&nbsp;15;26(12):i318-24. [</span><a href="https://sites.google.com/site/vibansal/software/crisp" style="line-height:1.5;background-color:rgb(255,255,255)">CRISP</a><span style="line-height:1.5;background-color:transparent">]</span></font></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div><font face="arial, sans-serif" size="2">Bansal V, Harismendy O, Tewhey R, Murray SS, Schork NJ, Topol EJ, Frazer KA. <a href="http://genome.cshlp.org/content/20/4/537.long">Accurate detection and genotyping of SNPs utilizing population sequencing data.</a>&nbsp;Genome Research 2010 Apr;20(4):537-45. <b>[<a href="http://www.biotechniques.com/news/Navigating-the-human-genome/biotechniques-316106.html">article in Biotechniques</a>] [<a href="http://www.genomeweb.com/sequencing/whats-your-genotype">article in genome technology</a>]</b></font></div>
<div><b><font face="arial, sans-serif" size="2"><br>
</font></b></div>
<div>
<div style="line-height:22.2222px"><font face="arial, sans-serif" size="2"><span style="line-height:22.2222px;background-color:transparent">Schork NJ, Bansal V.</span><span style="line-height:22.2222px;background-color:transparent">&nbsp;Not so lost in the genetic crowd.&nbsp;</span><span style="line-height:22.2222px;background-color:transparent">Nature Genetics 2009</span></font></div>
</div>
<div><span style="line-height:22.2222px;background-color:transparent"><font face="arial, sans-serif" size="2"><br>
</font></span></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div><b><font face="arial, sans-serif" size="2">2004-2008 (Ph.D. work)</font></b></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div><font face="arial, sans-serif" size="2">Bansal V, Bafna V. <a href="http://bioinformatics.oxfordjournals.org/content/24/16/i153.long">HapCUT: an efficient and accurate algorithm for the haplotype assembly problem.</a> Bioinformatics. 2008 Aug 15;24(16):i153-9.<br>
<br>
Bansal V, Halpern AL, Axelrod N, Bafna V. <a href="http://genome.cshlp.org/content/18/8/1336.long">An MCMC algorithm for haplotype assembly from whole-genome sequence data</a>. Genome Research 2008 Aug;18(8):1336-46.</font></div>
<div><font face="arial, sans-serif" size="2"><br>
</font></div>
<div><font face="arial, sans-serif" size="2">Levy S, Sutton G, Ng PC, Feuk L, Halpern AL, Walenz BP, Axelrod N, Huang J, Kirkness EF, Denisov G, Lin Y, MacDonald JR, Pang AW, Shago M, Stockwell TB, Tsiamouri A, Bafna V, Bansal V, Kravitz SA, Busam DA, Beeson KY, McIntosh TC, Remington KA, Abril JF, Gill J, Borman J, Rogers YH, Frazier ME, Scherer SW, Strausberg RL, Venter JC. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1964779/">The diploid genome sequence of an individual human.</a> PLoS Biol. 2007 Sep 4;5(10):e254.&nbsp;</font></div>
<div><font face="arial, sans-serif" size="2"><br>
</font>
<div><font face="arial, sans-serif" size="2"><span style="line-height:1.5;background-color:transparent">Gusfield D, Bansal V, Bafna V, Song YS. </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2581772/" style="line-height:1.5">A decomposition theory for phylogenetic networks and incompatible characters</a><span style="line-height:1.5;background-color:transparent">. J Comput Biol. 2007 Dec;14(10):1247-72.&nbsp;</span></font></div>
</div>
<div><font face="arial, sans-serif" size="2"><br>
Bansal V, Bashir A, Bafna V. <a href="http://genome.cshlp.org/content/17/2/219.long">Evidence for large inversion polymorphisms in the human genome from HapMap data</a>. Genome Research 2007 Feb;17(2):219-30. Epub 2006&nbsp;Dec 21. <br>
<br>
Bafna V, Bansal V. <a href="http://cseweb.ucsd.edu/users/vibansal/RecBounds/http://cseweb.ucsd.edu/users/vibansal/RecBounds/LowerBounds_JCB.pdf">Inference about recombination from haplotype data: lower bounds and recombination hotspots</a>. J Comput Biol. 2006 Mar;13(2):501-21. PubMed PMID: 16597254.<br>
<br>
</font>
<div><font face="arial, sans-serif" size="2"><a href="http://www.informatik.uni-trier.de/%7ELey///pers/hd/g/Gusfield:Dan.html">Dan Gusfield</a>, Vikas Bansal: <a href="http://csiflabs.cs.ucdavis.edu/~gusfield/gusfieldrecomb.pdf">A Fundamental Decomposition Theory for Phylogenetic Networks and Incompatible Characters</a>. <a href="http://www.informatik.uni-trier.de/%7ELey///db/conf/recomb/recomb2005.html#GusfieldB05">RECOMB 2005</a>: 217-232<br>
<br>
</font>
<div><font face="arial, sans-serif" size="2"><a href="http://www.informatik.uni-trier.de/%7ELey///pers/hd/b/Bafna:Vineet.html">Vineet Bafna</a>, <span>Vikas Bansal</span>: <span><a href="http://cseweb.ucsd.edu/users/vibansal/RecBounds/recomb2005_lowerbounds.pdf">Improved Recombination Lower Bounds for Haplotype Data.</a></span> <a href="http://www.informatik.uni-trier.de/%7ELey///db/conf/recomb/recomb2005.html#BafnaB05">RECOMB 2005</a>: 569-584<br>
<br>
Bafna V, Bansal V. <a href="http://cseweb.ucsd.edu/users/vibansal/RecBounds/tcbb04.pdf">The number of recombination events in a sample history: conflict graph and lower bounds</a>. IEEE/ACM Trans Comput Biol 
Bioinform. 2004 Apr-Jun;1(2):78-90. PubMed PMID: 17048383.<br>
</font></div>
</div>
<font face="arial, sans-serif" size="2"><br>
Vikas Bansal, <a href="http://www.informatik.uni-trier.de/%7ELey///pers/hd/h/Heide:Friedhelm_Meyer_auf_der.html">Friedhelm Meyer auf der Heide</a>, <a href="http://www.informatik.uni-trier.de/%7ELey///pers/hd/s/Sohler:Christian.html">Christian Sohler</a>: Labeling Smart Dust. <a href="http://www.informatik.uni-trier.de/%7ELey///db/conf/esa/esa2004.html#BansalHS04">ESA 2004</a>: 77-88 <br>
</font><br>
</div>
            
            </p>
            <!-- <p>Publications goes here .. </p> -->
        </header>
    </div>
    <!-- /.container -->

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

</body>

</html>
