<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Bansal Lab - Research</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/heroic-features.css" rel="stylesheet">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation -->
    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                 <a class="navbar-brand" href="index.html">Bansal Lab</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="research.html">Research</a>
                    </li>
                    <li>
                        <a href="publications.html">Publications</a>
                    </li>
                    <li>
                        <a href="people.html">People</a>
                    </li>
                    <li>
                        <a href="teaching.html">Teaching</a>
                    </li>
                    <li>
                        <a href="software.html">Software</a>
                    </li>
                    <li>
                        <a href="data.html">Data</a>
                    </li>
                    <li>
                        <a href="join.html">Join</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Page Content -->
    <div class="container">

        <!-- Jumbotron Header -->
        <header class="jumbotron hero-spacer">
            <p style="font-size:15px;font-weight:350;">
		    Research in the group is focused on developing computational methods and tools for variant calling in human genomes using high-throughput DNA sequencing technologies. 
		    Although human genome sequencing has advanced rapidly, DNA sequencing using short Illumina reads provides an incomplete picture of human genetic variation and it is difficult to identify certain types of variants. We focus on challenging variant types such as haplotypes and variants in repetitive regions and work with both short-read (Illumina) and long-read sequencing technologies (Pacific Biosciences and Oxford Nanopore). Our goal is to develop computational tools that can be used to enable discovery of new disease-associated variants and analyze human genetic variation.   
		    Two major topics of current research are: <br><br></p>
                
	    <div> <p style="font-size:15px;font-weight:350;"> 1. <b>Methods for accurate variant calling in repetitive regions of the genomes:</b> We are developing methods for the detection of variants in repetitive regions (segmental duplications) of the human genome using whole-genome Illumina sequencing as well as single-molecule long read sequencing technologies. This includes both copy number variants as well as small variants such as SNPs. These regions cover more than 5% of the genome and overlap more than <a href="https://pubmed.ncbi.nlm.nih.gov/27228465/">150 disease-associated genes</a> such as SMN1 (spinal muscular atrophy), STRC (hearing loss) and PMS2 (lynch syndrome). The goal is to analyze large-scale sequencing datasets and identify novel disease-associated variants in both rare and complex diseases.  </div> </p>
		
		    <div> <p style="font-size:15px;font-weight:350;"> 2. <b>Haplotype-based variant calling using long-read sequencing technologies:</b> We have developed a number of computational tools (e.g. HapCUT2) for reconstructing haplotypes in individual genomes that work with diverse sequencing technologies including linked-reads and long-read sequencing. Long-read sequencing technologies have the potential to overcome some of the key limitations of short-read sequencing, particular in long repetitive regions of the human genome, but require the development of new algorithms. We have previously developed computational methods for variant calling (Longshot, Nature Communications 2019) and read mapping in segmental duplications (<a href="https://academic.oup.com/nar/article/48/19/e114/5920427?login=true">Duplomap, Nucleic Acids Research 2020</a>) using long-read sequencing technologies. Our goal is to enable accurate variant calling using long read sequencing and haplotype-based probabilistic models, particularly in repetitive regions of the genome.  </div> </p>
				<p style="font-size:15px;font-weight:350;"><b>Selected publications:</b><br></p>

				<p style="font-size:15px;font-weight:350;">
		
			Prodanov T, Bansal V. <a href="http://github.com/tprodanov/parascopy">Robust and accurate estimation of paralog-specific copy number for duplicated genes using whole-genome sequencing</a> Nature Communications, in review <br> 


		Edge P, Bansal V. <a href="https://www.nature.com/articles/s41467-019-12493-y">Longshot: accurate variant calling in diploid genomes using single-molecule long read sequencing.</a> Nature Communications, Oct 2019. <br> 

		Bakhtiari M, Shleizer-Burko S, Gymrek M, <b>Bansal V</b>, Bafna V. <a href="https://genome.cshlp.org/content/28/11/1709">Targeted Genotyping of Variable Number Tandem Repeats with
				    adVNTR.</a> Genome Research, October 2018. <br>
		
Edge P, Bafna V, <b>Bansal V</b>. <a href="http://genome.cshlp.org/content/early/2016/12/09/gr.213462.116.abstract">HapCUT2: robust and accurate haplotype assembly for diverse sequencing technologies</a>, Genome Research, Dec 2016.<br>

			</p>	
 
            <!-- <p>Research goes here .. </p> -->
        </header>
	</h4>
    </div>
    <!-- /.container -->

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

</body>

</html>
