<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Bansal Lab - Software</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/heroic-features.css" rel="stylesheet">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation -->
    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                 <a class="navbar-brand" href="index.html">Bansal Lab</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="research.html">Research</a>
                    </li>
                    <li>
                        <a href="publications.html">Publications</a>
                    </li>
                    <li>
                        <a href="people.html">People</a>
                    </li>
                    <li>
                        <a href="software.html">Software</a>
                    </li>
                    <li>
                        <a href="data.html">Data</a>
                    </li>
                    <li>
                        <a href="join.html">Join</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Page Content -->
    <div class="container">

        <!-- Jumbotron Header -->
        <header class="jumbotron hero-spacer small">

            <h4><strong><a href="https://github.com/vibansal/hapcut2">HapCUT2</a>:</strong> new version of HapCUT that uses a likelihood based model for haplotype assembly and can model trans-errors in Hi-C data</h4>

            <h4><strong><a href="https://github.com/vibansal/PCRduplicates">PCRduplicates</a></strong> Computational method to estimate the PCR duplication rate in high-throughput DNA sequencing experiments</h4>

            <h4><strong><a href="https://github.com/vibansal/hapcut2">HapCUT</a></strong>: a program for haplotype assembly or haplotype phasing using sequence reads from a diploid genome. Source code is available from <a href="https://github.com/vibansal/hapcut">github</a></h4>


            <h4><strong><a href="https://bansal-lab.github.io/software/crisp.html">CRISP:</a></strong> program for detection of SNVs and short insertion/deletion variants from high-throughput sequencing of pooled DNA samples. Latest source code is available from github</h4>

            <h4><strong><a href="software/iadmix.html">iAdmix</a></strong> a program to estimate individual admixture proportions from DNA sequencing experiments (exome sequencing, targeted sequencing, low coverage whole-genome sequencing or genotyping). Source code is available from <a href="https://github.com/vibansal/ancestry">github</a> </h4>


            <h4><strong><a href="https://github.com/vibansal/hapcut/tree/master/scripts/hap-simulation">Hap-simulator</a></strong> a program for simulating reads for haplotype assembly to evaluate N50 metrics as a function of read length, insert size (paired-end reads), etc </h4>

            <br/>
            <br/>
            <h4><strong>Software that is no longer updated: </h4>

            <h4><strong>AncestryCLL</strong> a python program to estimate ancestry likelihoods using genotypes at Ancestry-Informative Markers and HapMap allele frequencies </h4>

            <h4><strong>piCALL</strong> a probabilistic method for detection of insertion/deletion polymorphisms (indels) from population-scale sequence data (use CRISP instead of piCALL) </h4>

            <h4><strong><a href="software/old/snipseq.html">SNIP-Seq</a></strong> one of the first methods designed to leverage sequence data from multiple individuals simultaneously to model site-specific error rates and detect SNPs </h4>

            <h4><strong>HASH</strong> implementation of an MCMC algorithm for haplotype assembly from whole-genome sequence data (use HapCUT instead)</h4>

            <h4><strong><a href="software/old/recombination.html">LUBOUND</a></strong> program to calculate lower bounds on the minimum number of recombination events in a sample history </h4>

        </header>
    </div>
    <!-- /.container -->

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

</body>

</html>
