<div style="font-size:13.6px;line-height:16px"><font size="3"><span style="font-family:arial,sans-serif"><span style="color:rgb(61,133,198)"><b>Haplotype assembly aka phasing from sequence reads&nbsp;</b></span></span></font></div>
<div style="font-size:13.6px;line-height:16px"><font size="3"><span style="font-family:arial,sans-serif"><span style="color:rgb(61,133,198)"><b><br>
</b></span></span></font></div>
<div style="font-size:13.6px;line-height:16px"><b style="font-size:medium;font-family:arial,sans-serif;line-height:20px"><span style="color:rgb(61,133,198)">
<div style="color:rgb(51,51,51);font-weight:normal;display:inline!important">
<div style="font-family:Lucida Grande,Lucida Sans,Verdana,Arial,sans-serif;font-size:13.6px;line-height:16px;display:inline!important"><span style="font-family:arial,sans-serif;font-size:small;line-height:20px">Even as the cost of DNA sequencing continues to decrease, our ability to extract information from genome seque</span></div>
</div>
</span></b><b style="font-size:medium;font-family:arial,sans-serif;line-height:20px"><span style="color:rgb(61,133,198)">
<div style="color:rgb(51,51,51);font-weight:normal;display:inline!important">
<div style="font-family:Lucida Grande,Lucida Sans,Verdana,Arial,sans-serif;font-size:13.6px;line-height:16px;display:inline!important"><span style="font-family:arial,sans-serif;font-size:small;line-height:20px">ncing data about human genetic variation is limited in many aspects. In particular, the short read lengths and the short fragment lengths used for genome sequencing limit the recovery of haplotype information. However, many findings indicate that haplotype information (the combination of alleles at variant sites on a single chromosome) is crucial to fully interpret the relationship between human DNA sequence and phenotype including disease. In 2007, we developed the HapCUT algorithm for haplotype assembly from whole-genome sequence data that has been demonstrated to be highly accurate and flexible enough to be applied to a wide variety of data types such as Hi-C data (Selvaraj et al. Nature Biotech 2013). Some projects that we are currently interested in:&nbsp;</span></div>
</div>
</span></b></div>
<div style="font-size:small">
<ul><li><font face="arial, sans-serif"><span style="line-height:20px">Development of algorithms for haplotype assembly from long-read data and Hi-C data&nbsp;</span></font></li>
<li><font face="arial, sans-serif"><span style="line-height:20px">Methods for reconstruction of haplotype fragments, structural variation and assembly of low copy repeats from dilution pool sequencing (pools with long DNA fragments)</span></font></li>
<li><font face="arial, sans-serif"><span style="line-height:20px">use of haplotype information to identify disease associated variants (e.g. compound heterozygotes in rare recessive disorders)&nbsp;</span></font></li></ul>
<div><font face="arial, sans-serif"><span style="line-height:20px"><br>
</span></font></div>
</div>
<div style="font-size:small"><font face="arial, sans-serif"><span style="line-height:20px">
<div style="text-align:center;margin-right:auto;margin-left:auto"></div>
</span></font></div>
<div style="font-size:13.6px;line-height:16px">
<div style="text-align:center;margin-right:auto;margin-left:auto"><a href="https://sites.google.com/site/vibansal/home/research/hapassembly.png?attredirects=0" imageanchor="1"><img border="0" height="295" src="research/hapassembly.png" width="400"></a><img border="0" height="272" src="research/hapcut.png" style="text-align:left;font-size:small;line-height:1.5" width="400"></div>
</div>
