<div>
<div><span style="font-family:times new roman,serif;font-size:small;line-height:1.5;background-color:transparent">In recent years, high-throughput sequencing technologies have transformed our understanding of human genetic variation by enabling the sequencing of individual human genomes as well as population-scale genome sequencing. Short insertions/deletions (indels) represent a functionally important and the second most frequent form of variation in the human genome. Indels have not received as much attention as single nucleotide variants (SNVs) and structural variants. Detection of indels from high-throughput sequence datasets is challenging and computational methods currently available for the detection of indels exhibit significantly lower sensitivity and specificity compared to available methods that can be used to identify single nucleotide variants. Novel computational methods that address the challenge presented by the detection and genotyping of&nbsp; indels are urgently needed. We are developing new methods for the detection of short indels from both individual and population-scale sequence datasets that will utilize information about sequence-context specific and sequencing platform specific indel error rates present in large-scale sequence datasets to generate accurate indel calls and genotypes.</span></div>
</div>
<div><br>
</div>
<div><span style="font-family:times new roman,serif"><font size="2"><b>Specific goals of this project include:</b><br>
<br>
</font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2">1. Design and implement a Bayesian method for detection and genotyping of short indels using multi-sample sequence data (whole-exome or whole-genome) &nbsp;<br>
</font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2">2. Develop algorithms for detection of large deletions and medium length insertions from whole-exome/whole-genome sequence data using a combination of split-read mapping, gapped alignment and realignment <br>
</font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2">3. Implement methods for indel calling in individual genomes using cross-site error rate modeling, estimation of context-specific error rates and haplotype information&nbsp;</font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2">4. Apply these methods to detect functional indels in exon-targeted sequencing studies, population sequencing studies and somatic indels in tumor-normal sequencing studies <br>
</font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2"><br>
<b style="background-color:transparent;line-height:1.5">References:</b></font></span></div>
<font size="2">
<div style="font-family:times new roman,serif"><span style="font-family:times new roman,serif"><font size="2"><br>
</font></span></div>
<div><span style="font-family:times new roman,serif;color:rgb(0,0,0)">Alastair Kilpatrick and Vikas Bansal.&nbsp;</span><span style="background-color:transparent"><font color="#000000" face="times new roman, serif">Detection and genotyping of short indels using sequence&nbsp;</font></span><span style="color:rgb(0,0,0);font-family:times new roman,serif;line-height:1.5;background-color:transparent">data from multiple samples. <a href="KilpatrickPosterISMB2016.pdf">ISMB 2016 poster</a> (also selected for oral presentation).</span></div>
<br>
<font face="times new roman, serif">Bansal V, Libiger O.&nbsp;</font><a href="http://bioinformatics.oxfordjournals.org/content/27/15/2047.abstract" rel="nofollow" style="font-family:times new roman,serif;background-color:rgb(255,255,255)">A probabilistic method for the detection and genotyping of small indels from population-scale sequence data</a><font face="times new roman, serif">. Bioinformatics. 2011 Aug 1;27(15):2047-53. Epub 2011 Jun 7. </font><b style="font-family:times new roman,serif"><br>
</b></font>
<div>
<div><span style="font-family:times new roman,serif"><font size="2"><br>
</font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2"><span style="color:black">Bansal V.&nbsp;</span><span style="color:black">Accurate genotyping
of INDELS from population-scale short read sequence data. Selected Talk at HitSeq 2010, Boston (<a href="HitSeq2010-indel-slides.pdf">slides</a>)</span></font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2"><font color="#000000"><br>
</font></font></span></div>
<div><span style="font-family:times new roman,serif"><font size="2"><br>
</font><br>
</span></div>
</div>
