<div><span style="color:rgb(61,133,198)"><font size="3"><b>Variant detection from pooled DNA sequencing data</b></font></span><br>
</div>
<div><br>

<div><font size="2"><span style="font-family:arial,sans-serif"><span>CRISP is a software program designed to 
detect SNPs and short indels from high-throughput sequencing of pooled 
DNA samples. CRISP has been primarily developed to analyze data from 
"artificial" DNA pools, i.e. pools generated by equi-molar pooling of 
DNA from multiple individual samples. CRISP leverages sequence data from
 multiple such pools to detect&nbsp;</span><span style="background-color:transparent;line-height:1.5">both rare and common variants. Note that the method is not designed for variant detection from a single pool. CRISP was developed for targeted disease association studies in humans but may work well for other applications. <br>
<br>
CRISP has been tested on a number of pooled targeted capture datasets as well as exome sequence data generated using the Illumina sequencing platform.&nbsp; </span></span><font style="background-color:transparent"><span style="font-family:arial,sans-serif"><span style="background-color:transparent;line-height:1.5">CRISP works directly with sorted BAM files and outputs a single VCF file with information about the variants and the genotypes (or allele frequencies) for each pool or sample. CRISP can identify both SNVs and short insertion/deletions (indels).&nbsp; </span></span></font><font style="background-color:transparent"><span style="font-family:arial,sans-serif"><span style="background-color:transparent;line-height:1.5">CRISP is implemented in C and uses the <a href="http://samtools.sourceforge.net/">SAMtools</a> API to read bam files.&nbsp;</span></span></font><span style="font-family:arial,sans-serif;line-height:19px;background-color:transparent">The latest version of CRISP should work well even for diploid genomes (pool size = 2).</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><span style="background-color:transparent;line-height:1.5"><br>
</span><span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881398/">A statistical method for the detection of variants from next-generation resequencing of DNA pools </a> Vikas Bansal. Bioinformatics, 2010.</span><br>
<span>
</span><br>
<br>
</span></font></div>
<font size="2"><span style="font-family:arial,sans-serif">The latest source code (updated 10/2014) can be downloaded and compiled from the <a href="https://github.com/vibansal/crisp/">Github repository</a>. <br>
</span></font>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
</span></font><font size="2"><span style="font-family:arial,sans-serif"><font size="2"><span style="font-family:arial,sans-serif">A binary for CRISP is available for <a href="crisp/CRISP-122713.tar.gz">download</a>. Unpack it using the "tar xvzf CRISP-xx.tar.gz" command. This will create a directory with the CRISP executable. The </span></font>CRISP executable can be run from the command line. CRISP requires multiple BAM files and an indexed reference fasta file to run, and outputs the variants and genotypes to a VCF file. Documentation on how to run CRISP and the input and ouput files is included in a README file in the tar package. <br>
</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
<br>
</span></font>
<div><font size="2"><span style="font-family:arial,sans-serif"><b>Latest source code (last updated 10/2014) is available from Github. CRISP can call 
variants from both pooled and unpooled (diploid) multi-sample datasets.</b>&nbsp;</span></font></div>
<font size="2"><span style="font-family:arial,sans-serif"><br>
<br>
</span></font><font size="2"><span style="font-family:arial,sans-serif"><font size="2"><span style="font-family:arial,sans-serif"><span>
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068187/">Efficient and cost effective population resequencing by pooling and in-solution hybridization.</a> V Bansal, R. Tewhey, EM Leproust, NJ Schork.</span>
<span> PLoS One,</span><span>&nbsp;2011&nbsp; <a href="http://polymorphism.scripps.edu/datasets/PooledSequencing/">[sequence data for download]</a><br><br>

 
&nbsp;</span></span></font>CRISP has been used to perform a pooled-sequencing based association study: Scott-Van Zeeland AA, Bloss C, Tewhey C, Bansal V, et al “<a href="http://www.nature.com/mp/journal/vaop/ncurrent/full/mp201391a.html">Evidence for the Role of EPHX2 gene variants in Anorexia Nervosa</a>”.&nbsp;&nbsp; Molecular&nbsp;
Psychiatry, 2013<br>
</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
</span></font></div>
<div><span style="font-family:arial,sans-serif;font-size:small;line-height:26.7px">Please send bug reports,&nbsp; comments, etc to&nbsp;</span><b style="font-family:arial,sans-serif;font-size:small;line-height:26.7px">vibansal AT cs.ucsd.edu&nbsp;</b></div>

 <hr>
 
 <div><span style="color:rgb(61,133,198)"><font size="2"><b>Updates to CRISP</b></font></span><br>
  <div>
<div>
 <br>
<div><b><span style="font-size:small">Dec 27 2013 <br>
<br>
</span></b></div>
<div><span style="font-size:small">A
 new version of CRISP has been uploaded (see downloads). CRISP can call 
variants from both pooled and unpooled (diploid) multi-sample datasets. </span><span style="font-size:small">The new features of CRISP are:<br>
<br>
</span></div>
<div><span style="font-size:small">1. EM algorithm to estimate the pooled genotypes (or allele counts) for each pool and the allele frequency jointly across all pools <br>
</span></div>
<div><span style="font-size:small">2. overlapping paired-end reads are processed correctly and bases that are read from both directions are counted only once <br>
</span></div>
<div><span style="font-size:small">3. CRISP works for diploid individuals as well as pooled data (multiple samples equi-molar pooled into a single pool) <br>
</span></div>

<div><b><span style="font-size:small"><br>
Nov 28 2013</span></b></div>
<div><b><span style="font-size:small"><br>
</span></b></div>
<div><span style="font-size:small">CRISP has been extended to estimate pooled genotypes using an EM algorithm, and can handle overlapping paired-end reads and multi-allelic variant sites. It is under active development and a stable version will be 
released soon.&nbsp;</span></div>
<span style="font-size:small">
<div><b><span style="font-size:small"><br>
</span></b></div>
<b>July 18 2012</b><br>
<br>
CRISP
 can now call variants directly from BAM files.&nbsp; The program (pre-compiled binaries for linux x86_64) is available for download. 
<div></div>
<div><span style="font-size:small"><br>
</span></div>
<b><span style="font-size:small">April 24 2012 </span></b><span style="font-size:small"><br>
<br>
A new version of CRISP has been implemented in C. This version is 
significantly faster than the previous python version, more accurate for
 both SNPs and insertions/deletions and also outputs the variants and 
the allele counts in the VCF format.<br>
<br>
</span></div>
<span style="font-size:small"><b>June 2010 <br>
<br>
</b></span></div>
<span style="font-size:small">The paper describing CRISP is published in Bioinformatics. The original software was implemented in Python. <br>
</span>
<div>
<div><span style="font-size:small"><br>
</span></div>
</div>
 
