<div><font size="2"><span style="font-family:arial,sans-serif"><span>CRISP is a software program designed to 
detect SNPs and short indels from high-throughput sequencing of pooled 
DNA samples. CRISP has been primarily developed to analyze data from 
"artificial" DNA pools, i.e. pools generated by equi-molar pooling of 
DNA from multiple individual samples. CRISP leverages sequence data from
 multiple such pools to detect&nbsp;</span><span style="background-color:transparent;line-height:1.5">both rare and common variants. Note that the method is not designed for variant detection from a single pool. CRISP was developed for targeted disease association studies in humans but may work well for other applications. <br>
<br>
CRISP has been tested on a number of pooled targeted capture datasets as well as exome sequence data generated using the Illumina sequencing platform.&nbsp; </span></span><font style="background-color:transparent"><span style="font-family:arial,sans-serif"><span style="background-color:transparent;line-height:1.5">CRISP works directly with sorted BAM files and outputs a single VCF file with information about the variants and the genotypes (or allele frequencies) for each pool or sample. CRISP can identify both SNVs and short insertion/deletions (indels).&nbsp; </span></span></font><font style="background-color:transparent"><span style="font-family:arial,sans-serif"><span style="background-color:transparent;line-height:1.5">CRISP is implemented in C and uses the <a href="http://samtools.sourceforge.net/">SAMtools</a> API to read bam files.&nbsp;</span></span></font><span style="font-family:arial,sans-serif;line-height:19px;background-color:transparent">The latest version of CRISP should work well even for diploid genomes (pool size = 2).</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><span style="background-color:transparent;line-height:1.5"><br>
</span><span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881398/">A statistical method for the detection of variants from next-generation resequencing of DNA pools </a> Vikas Bansal. Bioinformatics, 2010.</span><br>
<span>
</span><br>
<br>
</span></font></div>
<font size="2"><span style="font-family:arial,sans-serif">The current version of CRISP (CRISP-122713.tar.gz) can be downloaded from the attachments below.&nbsp; The latest source code (updated 10/2014) can be downloaded and compiled from the <a href="https://github.com/vibansal/crisp/">Github repository</a>. <br>
</span></font>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
</span></font><font size="2"><span style="font-family:arial,sans-serif"><font size="2"><span style="font-family:arial,sans-serif">Download the tar-ball from the attachments and unpack it using the "tar xvzf CRISP-xx.tar.gz" command. This will create a directory with the CRISP executable. The </span></font>CRISP executable can be run from the command line. CRISP requires multiple BAM files and an indexed reference fasta file to run, and outputs the variants and genotypes to a VCF file. Documentation on how to run CRISP and the input and ouput files is included in a README file in the tar package. <br>
</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
<br>
</span></font>
<div><font size="2"><span style="font-family:arial,sans-serif"><b>Latest source code (last updated 10/2014) is available from Github. CRISP can call 
variants from both pooled and unpooled (diploid) multi-sample datasets.</b>&nbsp;</span></font></div>
<font size="2"><span style="font-family:arial,sans-serif"><br>
<br>
</span></font><font size="2"><span style="font-family:arial,sans-serif"><font size="2"><span style="font-family:arial,sans-serif"><span>
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068187/">Efficient and cost effective population resequencing by pooling and in-solution hybridization </a> V Bansal, R. Tewhey, EM Leproust, NJ Schork.</span>


<span> PLoS One,</span><span>&nbsp;2011&nbsp; <a href="http://polymorphism.scripps.edu/datasets/PooledSequencing/">[sequence data for download]</a><br>
&nbsp;</span></span></font>CRISP has been used to perform a pooled-sequencing based association study: Scott-Van Zeeland AA, Bloss C, Tewhey C, Bansal V, et al “<a href="http://www.nature.com/mp/journal/vaop/ncurrent/full/mp201391a.html">Evidence for the Role of EPHX2 gene variants in Anorexia Nervosa</a>”.&nbsp;&nbsp; Molecular&nbsp;
Psychiatry, 2013<br>
</span></font></div>
<div><font size="2"><span style="font-family:arial,sans-serif"><br>
</span></font></div>
<div><span style="font-family:arial,sans-serif;font-size:small;line-height:26.7px">Please send bug reports,&nbsp; comments, etc to&nbsp;</span><b style="font-family:arial,sans-serif;font-size:small;line-height:26.7px">vibansal AT cs.ucsd.edu&nbsp;</b></div>
